services/fhir.ts

```diff
@@ -160,17 +236,27 @@ export async function saveFHIRResources<R extends AidboxResource>(
 }
 
 export async function patchFHIRResource<R extends AidboxResource>(
-    resource: Partial<R> & Required<Pick<R, 'id' | 'resourceType'>>
+    resource: Partial<R> & Required<Pick<R, 'id' | 'resourceType'>>,
+    searchParams?: SearchParams
 ): Promise<RemoteDataResult<R>> {
-    return service({
+    return service(patch(resource, searchParams));
+}
+
+export function patch<R extends AidboxResource>(
+    resource: Partial<R> & Required<Pick<R, 'id' | 'resourceType'>>,
+    searchParams?: SearchParams
+): AxiosRequestConfig {
+    return {
         method: 'PATCH',
         data: resource,
         url: `/${resource.resourceType}/${resource.id}`,
```

Comment:
Let's change url: if we have searchParams - it should be /resourceType, otherwise /resourceType/id. And throw an exception if searchParams and id both specified.